### ISSUE TITLE
[Suggestion]: Maybe a rework/overhaul of how clouds move visually?

### ISSUE THREAD

**Issue by Luke7381** [nonamecrackers2/simple-clouds#69]
### Guidelines

- [X] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Description of feature request

Me and some others kinda find it weird and distracting of how the voxels just disappear and reappear when a cloud is moving, it almost looks like "glitching" somewhat. Its not to bad when it comes to cumulonimbus clouds but its very jarring on small clouds. Is there plans on changing the way they move visually?

Here is a reddit post about this mod. Lots of the replies have the same opinion as me.
https://www.reddit.com/r/feedthebeast/comments/1gcfso4/simple_clouds_mod_is_amazing/

**Comment by nonamecrackers2** [comment]
Yep I definitely agree. I plan on using transparency to help make cube transitions a bit better and to help with the visual appeal a bit more.

**Comment by nonamecrackers2** [comment]
Will be fixed using transparency in the next release
